spec/models/user_spec.rb

```diff
@@ -709,4 +709,21 @@
       expect(User.find_by_manager_login("admin_user_#{user.id}")).to eq user
     end
   end
+
+  describe "#after_restore" do
+    it "restore all previous hidden user content" do
+      user                  = create(:user, :hidden)
+      debate                = create(:debate, :hidden, author: user)
+      comment               = create(:comment, :hidden, author: user)
+      proposal              = create(:proposal, :hidden, author: user)
+      budget_investment     = create(:budget_investment, :hidden, author: user)
```

Comment:
Although we don't do it very often, aligning code is OK. If possible, align it in a way so there's only one space between the longest column on the left and the equals sign.

I guess if you add a `proposal_notification` variable all these lines will remain exactly the way they are now, though :smile:.